CONTRIBUTING.md

```diff
@@ -1,19 +1,84 @@
+    - You should rather leave stuff out than include too much.
+- Well-documented (wiki, descriptive README.md, javadocs or command-line help available).
+- Open-source with a [free license](https://www.gnu.org/licenses/license-list.html).
+- Actively maintained (no dead projects!).
+
+---
 
-## Adding a new entry to the list
+<h2 align="center">Entry Additions</h2>
 
 1. Make sure the entry is [eligible](#eligibility) for this list.
-2. Click on the pen icon at the top of the README file.
-3. Add the entry at the alphabetically right position. It should use the following convention with a [SPDX License Identifier](https://spdx.org/licenses):
-```markdown
-[Project Name](Source link) - Description. ([Wiki](Wiki Link)) `SPDX License Identifier`
-```
-5. At the bottom of the page state what you added.
+2. Click on the pen icon at the top of the [README](README.md).
+3. Add the entry such that it respects alphabetical order within its parent category. It should use the following convention with a [SPDX License Identifier](https://spdx.org/licenses):
+
+<div align="center">
+  <table>
+    <tr>
+      <th>Wiki Available</th> <th>Markdown Format</th>
+    </tr>
+    <tr>
+      <td align="center">✔️</td>
+      <td align="center">
+        <br />
+        <pre lang="markdown">[Project Name](Source Link) - Description. ([Wiki](Wiki Link)) `SPDX License Identifier`</pre>
+      </td>
+    </tr>
+    <tr>
+      <td align="center">❌</td>
+      <td align="center">
+        <br />
+        <pre lang="markdown">[Project Name](Source Link) - Description. `SPDX License Identifier`</pre>
+      </td>
+    </tr>
+  </table>
+</div>
+
+5. In the description box at the bottom of the page, state what you added.
+6. Propose the changes and open a pull request.
+
+---
+
+<h2 align="center">Category & Subcategory Additions</h2>
+
+<p align="center"><i>When creating new <a href="#entry-additions">entries</a>, it may be necessary to create a new category to logcally sort the entry under.</i></p>
+
+1. Make sure there is no existing category underneath one of the [main categories](README.md#contents) that could logically contain the entry.
+2. Click on the pen icon at the top of the [README](README.md).
+3. Add the category such that it respects alphabetical order amongst the other subcategories. It should use the following convention:
+
+<div align="center">
+  <table>
+    <tr>
+      <th colspan=2>Category Depth</th> <th>Markdown Format</th>
+    </tr>
+    <tr>
+      <td align="center"><b>1</b></td>
+      <td align="center"><b>Child</b></td>
+      <td align="center">
+        <br />
+        <pre lang="markdown">### Library Category Name</pre>
+      </td>
+    </tr>
+    <tr>
+      <td align="center"><b>2</b></td>
+      <td align="center"><b>Grandchild<br />(Child of Child)</b></td>
+      <td align="center">
+        <br />
+        <pre lang="markdown">#### Library Subcategory Name</pre>
+      </td>
+    </tr>
+  </table>
```

Comment:
Same here, this should be possible in GitHub flavored Markdown.

Reply:
Once again, GFM-style tables don't work with the code blocks inside in the same way (at least, as far as I could tell).

Reply:
See: https://stackoverflow.com/a/46648510